go/tasks/config_load_test.go

```diff
@@ -73,6 +73,44 @@ func TestLoadConfig(t *testing.T) {
 		assert.Equal(t, map[string]string{"x/interruptible": "true"}, k8sConfig.InterruptibleNodeSelector)
 		assert.Equal(t, "x/flyte", k8sConfig.InterruptibleTolerations[0].Key)
 		assert.Equal(t, "interruptible", k8sConfig.InterruptibleTolerations[0].Value)
+		assert.Contains(t, k8sConfig.ArchitectureNodeSelector, "amd64")
+		assert.Equal(t, map[string]string{"x/architecture": "amd64"}, k8sConfig.ArchitectureNodeSelector["amd64"])
+		assert.Contains(t, k8sConfig.ArchitectureTolerations, "amd64")
+		assert.Equal(t, "x/flyte", k8sConfig.ArchitectureTolerations["amd64"][0].Key)
+		assert.Equal(t, "amd64", k8sConfig.ArchitectureTolerations["amd64"][0].Value)
+		assert.NotNil(t, k8sConfig.DefaultPodSecurityContext)
+		assert.NotNil(t, k8sConfig.DefaultPodSecurityContext.FSGroup)
+		assert.Equal(t, *k8sConfig.DefaultPodSecurityContext.FSGroup, int64(2000))
+		assert.NotNil(t, k8sConfig.DefaultPodSecurityContext.RunAsGroup)
+		assert.Equal(t, *k8sConfig.DefaultPodSecurityContext.RunAsGroup, int64(3000))
```

Comment:
looks like these are unrelated to this change ?

Reply:
Yep, didn't see that one